projects/storefrontlib/src/lib/global-message/facade/global-message.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { Observable } from 'rxjs';
+import { filter } from 'rxjs/operators';
+
+import * as fromStore from '../store';
+import { GlobalMessage, GlobalMessageType } from '../models/message.model';
+
+@Injectable()
+export class GlobalMessageService {
+  readonly messages$: Observable<any> = this.store.pipe(
+    select(fromStore.getGlobalMessagesEntities),
+    filter(data => data !== undefined)
+  );
```

Comment:
Maybe there is something I don't understand about the code, but it seems more consistent to create a facade method ( getMessages() perhaps ) to return the messages instead of declaring a public property.

Reply:
getMessages() and this messages$ are the same thing. In all other facades which were merged in develop branch, we used this kind of readonly variables.

Reply:
If getMessages() and this messages$ are the same thing, it would be more consistent for the user of the facades if the facade exposed only functions. Mixing variables and functions for no reason results in potentially confusing api.

Reply:
We agreed to leave it as is for now and address this in GH-272